.ci/eclipse-compiler-javac.sh

```diff
@@ -9,8 +9,11 @@ fi
 
 JAVA_RELEASE=${2:-1.8}
 
-ECJ_JAR="ecj-4.17.jar"
-ECJ_MAVEN_VERSION="R-4.17-202009021800"
+# Eclipse releases every 13 weeks: https://wiki.eclipse.org/SimRel/Simultaneous_Release_Cycle_FAQ
+# After that these variables should be updated.
+ECJ_RELEASE=R4_20
```

Comment:
maybe there is a way to resolve `latest` to something like `4.20` and then infer all there variables?